src/demo/index.html

```diff
@@ -437,6 +473,15 @@ <h1>Symphony Electron API Demo</h1>
             }
         });
 
+        /**
+         * Log retriever
+         */
+        if (window.ssf) {
```

Comment:
Do we need to implement both ways of sending logs? Both the ssf javascript method, and the postMessage() method? I thong we were not to use the old one any more, because it doesn't work with different contexts. Is it not enough to implement the postMessage() way? Will anyone use the other one?

Reply:
I'm not sure, my understanding was we needed both for compatibility. Maybe @KiranNiranjan would have more details.

Reply:
Yes, we definitely need to support both as currently, 

1. We have this feature configurable so, this should work if users turn off context isolation.
2. There will be a new API that we are going to be using in the latest versions of Electron `6.x`+ that requires the support for SSF APIs